app/Base/Controllers/DataTableController.php

```diff
@@ -112,6 +119,11 @@ class='img-responsive'
                 return $model->$boolean_column == true ? trans("admin.fields.yes") : trans("admin.fields.no");
             });
         }
+        foreach ($this->count_columns as $count_column) {
+            $datatables = $datatables->editColumn($count_column, function ($model) use ($count_column) {
+                return $model->$count_column == true ? $model->$count_column->count() : 0;
```

Comment:
I think it would be better to check the count or emptiness here?

```php
return count($model->$count_column) ? $model->$count_column->count() : 0;
```